client/src/controllers/scoresheet/scoresheet_controller.js

```diff
@@ -87,13 +97,24 @@ class ScoresheetController {
         if (scoreDiff !== 0 && isFinite(scoreDiff)) {
             this.showScoreDiffAnimation(scoreDiff)
         }
+
+        if (this.match && this.match.indexOf('✔') > -1) {

```

Comment:
Again, if you use a seperate value in the `select` on the HTML, you could keep the match object or match id, and here just use the `match.complete` property